src/tools/genpolicy/genpolicy-settings.json

```diff
@@ -277,7 +277,7 @@
         ]
     },
     "kata_config": {
-        "confidential_guest": false
+        "confidential_guest": true
```

Comment:
Given that https://github.com/microsoft/kata-containers/pull/171/commits/3cc0745d6c10bd8afb48d14673af5fcfc4d9905d breaks the functionality of the MSFT fork, I would remove the second part of https://github.com/microsoft/kata-containers/pull/171/commits/3cc0745d6c10bd8afb48d14673af5fcfc4d9905d (including its second commit comment) instead of adding https://github.com/microsoft/kata-containers/pull/171/commits/d2df44913e2eaab15ec93bdfd61c0a9dbcae9e2b. 

But, it's OK with me in case you prefer to keep this additional commit.